joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -325,6 +362,33 @@ JointTrajectoryController::halt()
   set_op_mode(hardware_interface::OperationMode::ACTIVE);
 }
 
+void JointTrajectoryController::publishState()
+{
+  if (state_publisher_period_.seconds() > 0.0 &&
+    (last_state_publish_time_ + state_publisher_period_) < lifecycle_node_->now())
+  {
+    if (state_publisher_ && state_publisher_->trylock()) {
+      last_state_publish_time_ = lifecycle_node_->now();
+
+      state_publisher_->msg_.header.stamp = last_state_publish_time_;
+      // TODO(ddengster): The following code is ported from ros_controllers.
+      // Port it when we're ready to put in the other trajectory code.
+
+      // state_publisher_->msg_.header.stamp          = time_data_.readFromRT()->time;
+      // state_publisher_->msg_.desired.positions     = desired_state_.position;
+      // state_publisher_->msg_.desired.velocities    = desired_state_.velocity;
+      // state_publisher_->msg_.desired.accelerations = desired_state_.acceleration;
+      // state_publisher_->msg_.actual.positions      = current_state_.position;
+      // state_publisher_->msg_.actual.velocities     = current_state_.velocity;
+      // state_publisher_->msg_.error.positions       = state_error_.position;
+      // state_publisher_->msg_.error.velocities      = state_error_.velocity;
```

Comment:
Do you want to open a follow-up ticket for this? I don't see a big point in having commented code checked in.

Reply:
I will likely put the code in with the action server; it's the part I need to get the numbers correct for my purposes. https://github.com/ros-controls/ros2_controllers/pull/26

Reply:
Which is perfect, please remove the commented code from this PR.